### ISSUE TITLE
False positives?

### ISSUE THREAD

**Issue by laryn** [MegaphoneJon/org.cpehn.subscriptionhistory#2]
The group that I installed this for is reporting false positives -- ie. people who were added to the group at one point, then were removed, but are still showing up in reports for people who are in the group.

Ideally when searching on a group with the status "Added", the report would only show people who are still in the group, not people who were in the group at one point but may not be anymore. I'm going to see if I can figure anything out but wanted to flag this here as well in case I can't come up with anything.

**Comment by eth4ever** [comment]
@laryn I don't think this is the intention of this extension. When you look up the people that have been added to your group, you get a list of every one that has been added over time to this group. Wheater or not they have been remove from the group again.
If you want only the people that are member to that group right now, just search for the group memebers.

**Comment by laryn** [comment]
Workaround for those who need to find out who subscribed during a specific timeframe, and remains a subscriber currently:

- Create a group to put the results of the subscription history date range report/filter (specify the group, the date range, and the status "Added").
- Use search builder to find contacts that are in this new group as well as the group you are filtering on for subscription history

It's a few extra steps but this way you get anyone who is currently a subscriber and who subscribed in the specific timeframe.

**Comment by masetto** [comment]
@MegaphoneJon is this extension still maintaned? Or are there other alternatives?
Because this issue is very important to fix because I tihink it should filter on the last subscription state.

**Comment by MegaphoneJon** [comment]
@masetto The extension works this way by design.  I would imagine what you want would involve adding a new filter "current group members only" rather than modifying the existing behavior.

I'm closing this, but will accept patches or sponsorship to implement this.
